recipes/simpleitk/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set version = "0.9.1" %}
+
+package:
+  name: simpleitk
+  version: "0.9.1"
+
+source:
+  git_url: http://itk.org/SimpleITK.git
+  git_tag:  v{{ version }}
+
+build:
+    number: 0
+
+requirements:
+  build:
+    - python
+    - cmake    >=2.8.11
+    - libtiff  4.0.*  # [not win]
+    - libpng   1.6.*  # [not win]
+    - jpeg     8*  # [not win]
+    - zlib     1.*  # [not win]
```

Comment:
Please update to match these [pinnings]( https://github.com/conda-forge/conda-forge.github.io/blob/df07f63f6c7729d1bc96177f7de85b991483e819/scripts/pin_the_slow_way.py#L32-L48 ).